opentelemetry-user-events-metrics/src/exporter/mod.rs

```diff
@@ -0,0 +1,96 @@
+use crate::transform::transform_resource_metrics;
+use async_trait::async_trait;
+
+use opentelemetry_api::metrics::{MetricsError, Result};
+use opentelemetry_sdk::metrics::{
+    data::{ResourceMetrics, Temporality},
+    exporter::PushMetricsExporter,
+    reader::{AggregationSelector, TemporalitySelector},
+    Aggregation, InstrumentKind,
+};
+
+use crate::tracepoint;
+use prost::Message;
+use std::fmt::{Debug, Formatter};
+
+#[derive(Clone, Copy)]
+pub struct MetricsExporter {}
+
+impl MetricsExporter {
+    pub fn new() -> MetricsExporter {
+        // This is unsafe because if the code is used in a shared object (DLL),
+        // the event MUST be unregistered before the shared object unloads.
+        unsafe {
+            let result = tracepoint::register();
```

Comment:
What is the tracepoint name associated with this user_event - is it unique for each exporter instance, and can be enabled/disabled separately ?